notify/receivers.go

```diff
@@ -47,12 +71,12 @@ type InvalidReceiverError struct {
 }
 
 type GrafanaReceiver struct {
-	UID                   string                 `json:"uid"`
-	Name                  string                 `json:"name"`
-	Type                  string                 `json:"type"`
-	DisableResolveMessage bool                   `json:"disableResolveMessage"`
-	Settings              map[string]interface{} `json:"settings"`
-	SecureSettings        map[string]string      `json:"secureSettings"`
+	UID                   string            `json:"uid"`
+	Name                  string            `json:"name"`
+	Type                  string            `json:"type"`
+	DisableResolveMessage bool              `json:"disableResolveMessage"`
+	Settings              json.RawMessage   `json:"settings"`
```

Comment:
this will actually help in moving forward refactoring in Grafana, because we have to do conversions back and forth for that field.